docker/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Run tests and make jar
+# Uses docker.
+
+set -eux
+
+# check if mysql is running
+mysql_running=$(docker inspect -f {{.State.Running}} mysql)
+if ! $mysql_running ; then
```

Comment:
The check for `Running` is not sufficient.  An "exited" mysql container on my Mac caused this to fail.